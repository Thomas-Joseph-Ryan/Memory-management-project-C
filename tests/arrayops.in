set a 1 2 3 4 5
rev a
get a
append a 5 5 6 7 6 6 5
get a
uniq a
get a
sort a
get a
uniq a
get a