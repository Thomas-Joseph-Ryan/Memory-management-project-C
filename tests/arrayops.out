> ok

> ok

> [5 4 3 2 1]

> ok

> [5 4 3 2 1 5 5 6 7 6 6 5]

> ok

> [5 4 3 2 1 5 6 7 6 5]

> ok

> [1 2 3 4 5 5 5 6 6 7]

> ok

> [1 2 3 4 5 6 7]

> 
bye
